templates/sshd_config.erb

```diff
@@ -17,7 +17,11 @@
 Port <%= @sshd_config_port %>
 #Protocol 2,1
 Protocol 2
+<% if @sshd_addressfamily != nil -%>
+AddressFamily <%= @sshd_addressfamily %>
+<% else -%>
 #AddressFamily any
```

Comment:
this should always be present as we made a design choice to show the defaults, which aids in debugging a broken config